docs/warnings/invalid-element-type.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Invalid Element Type Warning
+layout: single
+permalink: warnings/invalid-element-type.html
+---
+
+You probably came here because your code is trying to create a ReactElement with an invalid type using JSX or the `React.createElement` API. This usually happens when you have an invalid import statement.
+
+`React.createElement` requires an argument of type `string` (e.g. 'div', 'span'), or a `ReactClass`/`React.Component`. It cannot be of type `number`, `boolean`, `undefined` or `null`. See the documentation of this API: [https://facebook.github.io/react/docs/top-level-api.html](https://facebook.github.io/react/docs/top-level-api.html)
```

Comment:
I was referring to the `createElement` API. Not sure why I didn't see the hash button for the permalink, will fix!